app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -130,53 +130,45 @@ export class FollowTags extends Component {
               {this.renderFollowCount()}
             </header>
             <div data-testid="onboarding-tags" className="onboarding-tags">
-              {allTags.map((tag) => (
-                <div
-                  className={`onboarding-tags__item ${
-                    selectedTags.includes(tag) &&
-                    'onboarding-tags__item--selected'
-                  }`}
-                  style={{
-                    boxShadow: selectedTags.includes(tag)
-                      ? `inset 0 0 0 100px ${tag.bg_color_hex}`
-                      : `inset 0 0 0 2px ${tag.bg_color_hex}`,
-                    color: selectedTags.includes(tag) ? tag.text_color_hex : '',
-                  }}
-                >
-                  <div className="onboarding-tags__item__inner">
-                    #{tag.name}
-                    <button
-                      type="button"
-                      onClick={() => this.handleClick(tag)}
-                      className={`onboarding-tags__button ${
-                        selectedTags.includes(tag) &&
-                        'onboarding-tags__button--selected'
-                      }`}
-                      style={{
-                        backgroundColor: selectedTags.includes(tag)
-                          ? tag.text_color_hex
-                          : tag.bg_color_hex,
-                        color: selectedTags.includes(tag)
-                          ? tag.bg_color_hex
-                          : tag.text_color_hex,
-                      }}
-                    >
-                      {selectedTags.includes(tag) ? (
-                        <span>
-                          <span className="onboarding-tags__button-default">
-                            ✓ Following
```

Comment:
nitpick (blocking) - just one small thing - the checkmark that previously showed when you are following a tag is no longer appearing. Previously it was added like this, but I think it'd be an improvement to use the checkmark svg in combination with the `crayons-btn--icon-left` class ([like this example in storybook](https://storybook.forem.com/?path=/story/components-buttons-html--icon-left))

Reply:
Thanks for you time @aitchiss I have made the changes

<img width="1680" alt="Screenshot 2021-05-03 at 7 02 21 PM" src="https://user-images.githubusercontent.com/8092120/116882545-3449c180-ac42-11eb-949d-9d04ca161be8.png">